webapp/channels/src/tests/react_testing_utils.tsx

```diff
@@ -41,6 +41,19 @@ export const renderWithIntlAndStore = (component: React.ReactNode | React.ReactN
     );
 };
 
+export const renderWithRealStore = (component: React.ReactNode | React.ReactNodeArray, initialState: DeepPartial<GlobalState> = {}, locale = 'en', divContainer?: HTMLDivElement) => {
```

Comment:
Why not use `renderWithFullContext` here instead? I know I'm the one that added our third `renderWithX` helper, but I was hoping to have our code converge on fewer of those instead of adding more over time

Reply:
Not sure why I did not use `renderWithRealStore` at the beginning, but now it seems to be working just fine, so... `renderWithRealStore` removed :P